README.md

```diff
@@ -120,7 +120,8 @@ class Example extends React.Component {
                    onSuggestionsUpdateRequested={this.onSuggestionsUpdateRequested}
                    getSuggestionValue={getSuggestionValue}
                    renderSuggestion={renderSuggestion}
-                   inputProps={inputProps} />
+                   inputProps={inputProps}
+                    selectFirstSuggestion={false} />
```

Comment:
The example in `redux-devtools` is much smaller, so adding one more prop there makes sense. Here, I feel like the Basic Usage example is already too big, which can be a barrier for some people. Personally, if I wanted to highlight the first suggestion and couldn't immediately see a prop for it, I'd click Cmd+F and type "first", which will reveal the desired prop.